metecho/api/models.py

```diff
@@ -775,6 +780,26 @@ def finalize_submit_review(
             if delete_org and deletable_org:
                 org.queue_delete(originating_user_id=originating_user_id)
 
+    def try_send_assignment_emails(self):
+        for type_ in ("dev", "qa"):
+            if self.tracker.has_changed(f"assigned_{type_}"):
```

Comment:
@wlonk I'm curious about the decision to use a `tracker` here, when we're already doing these exact same checks in the serializer `update` method (https://github.com/oddbird/Metecho/pull/310/files#diff-b5db231bd778f94eb85ba5aefdfcbb0eR351-R358).

Reply:
Ok. I think I'm generally nervous about trackers because of the bugs we were hitting early on, when trackers were used and models were being passed in and out of jobs. I _think_ we maybe resolved that otherwise, but I was mostly turned off by them.

Reply:
Mmm, a fair point. It's all sans-tracker in the serializer now in any case.